licenseApp.js

```diff
@@ -15,16 +15,41 @@ function serverListen(server) {
     }
 }
 
+/* In practice, this service is accessed as http://localhost:3010/,
+ * but why not permit access as http://[::1]:3010 too?
```

Comment:
Might be worthwhile linking to your analysis in https://phabricator.endlessm.com/T20857 here.

Reply:
Done